lib/text_view.rb

```diff
@@ -16,4 +16,11 @@ def render_coordinates_error(coordinates)
 
     ERROR
   end
+
+  def render_computer_thinking
+    output.puts(<<~BRAINS)
```

Comment:
Maybe it should also sleep for a bit too! LOL. But that wouldn't belong here because ... this is a printer.

Reply:
I definitely want to do this, but I also don't want the test suite to slow down because of it. I'm thinking of doing an addition at the end where I add a way to control the sleep time, so the tests can make it zero.